rulesets/main.ts

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env -S deno run --allow-net
+
+import { Octokit } from "https://esm.sh/@octokit/core@6.1.2";
+import { paginateGraphQL } from "https://esm.sh/@octokit/plugin-paginate-graphql@5.2.2";
+import { z } from "https://esm.sh/zod@3.23.8";
+
+if (Deno.args.length < 1) {
+  console.error(
+    "Please provide your GitHub personal access token as an argument."
+  );
+  Deno.exit(1);
+}
```

Comment:
This should probably be read via https://docs.deno.com/runtime/manual/basics/env_variables/ instead of as an arg. This is how it is done in userstyles Octokit stuff as well FYI.